### DISCUSSION TITLE
How to configure `FILES_URL`

### DISCUSSION THREAD

**Discussion by CXwudi** [langgenius/dify#7527] [Help] [answered]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I would like to use set `MULTIMODAL_SEND_IMAGE_FORMAT=url`, however based on the documentation:

https://github.com/langgenius/dify/blob/6f968bafb2b0038253de1c605fae6e8db864fd70/docker/.env.example#L432
https://github.com/langgenius/dify/blob/6f968bafb2b0038253de1c605fae6e8db864fd70/docker/.env.example#L39-L42

It is unclear how to configure the `FILES_URL`. Is it the URL of a photo hosting site (like sm.ms)? Or is it a public URL of a s3 bucket?

**Comment by crazywoola** [comment, marked as answer]
> Is it the URL of a photo hosting site (like sm.ms)? Or is it a public URL of a s3 bucket?

For example `https://upload.dify.ai`, you can check the output image's url, should be something like this.

**Reply by CXwudi** [reply]
Can I use the same domain I used for dify deployment. For example I deployed dify using the docker compose way and I access my webui through `https://dify.example.com`. The can I put  `https://dify.example.com` as `FILES_URL`

**Reply by CXwudi** [reply]
Confirmed that I can just use the domain I used for accessing Dify in `FILES_URL`
